src/main/java/views/LeagueSimulatorPane.java

```diff
@@ -0,0 +1,220 @@
+        matchBox8.getChildren().addAll(match8, comboBox15, comboBox16);
+
+        matches1and2.getChildren().addAll(matchBox1,matchBox2);
+        matches3and4.getChildren().addAll(matchBox3,matchBox4);
+        matches5and6.getChildren().addAll(matchBox5,matchBox6);
+        matches7and8.getChildren().addAll(matchBox7,matchBox8);
+
+        matchesContainer.getChildren().addAll(matches1and2,matches3and4,matches5and6,matches7and8);
+
+        this.setLeft(leftContainer);
+        this.setCenter(centerContainer);
+    }
+
+    @Override
+    void styleComponents() {
+        leftContainer.setStyle(" -fx-padding: 20 20 20 20;\n" +
+                "        -fx-alignment: top-center;\n" +
+                "        -fx-background-color: #ccc;\n" +
+                "        -fx-spacing: 15;");
+        leftContainer.setAlignment(Pos.CENTER);
+
+        //centerContainer.setAlignment(Pos.CENTER);
+        centerContainer.setStyle("-fx-alignment: top-left;\n"+
+                " -fx-padding: 20 20 20 20;\n"+
+                "        -fx-spacing: 15;");
+
+        matchesContainer.setAlignment(Pos.CENTER);
+
+        title.setStyle("-fx-font-size: 20;");
+
+        String bigButton = "    -fx-font-size: 15;\n" +
+                "    -fx-font-family: sans-serif;\n" +
+                "    -fx-pref-width: 200;\n" +
+                "    -fx-pref-height: 40;\n" +
+                "    -fx-background-radius: 16px;";
+
+        returnButton.setStyle(bigButton);
+        playButton.setStyle(bigButton);
+    }
+
+    void createLeagueSchedule() throws Exception{
+
+        Queue<Team> schedule = new LinkedList<>();
+        String[] teamNames = {comboBox1.getSelectionModel().getSelectedItem(),
+                comboBox2.getSelectionModel().getSelectedItem(),
+                comboBox3.getSelectionModel().getSelectedItem(),
+                comboBox4.getSelectionModel().getSelectedItem(),
+                comboBox5.getSelectionModel().getSelectedItem(),
+                comboBox6.getSelectionModel().getSelectedItem(),
+                comboBox7.getSelectionModel().getSelectedItem(),
+                comboBox8.getSelectionModel().getSelectedItem(),
+                comboBox9.getSelectionModel().getSelectedItem(),
+                comboBox10.getSelectionModel().getSelectedItem(),
+                comboBox11.getSelectionModel().getSelectedItem(),
+                comboBox12.getSelectionModel().getSelectedItem(),
+                comboBox13.getSelectionModel().getSelectedItem(),
+                comboBox14.getSelectionModel().getSelectedItem(),
+                comboBox15.getSelectionModel().getSelectedItem(),
+                comboBox16.getSelectionModel().getSelectedItem()};
+
+        for(int i = 0; i < teamNames.length; i++){
+            if(teamNames[i] != null)
+                schedule.add(GAME.getTeamByName(teamNames[i]));
+            else
+                throw new Exception("Field empty");
+        }
+
+        GAME.createLeague(schedule);
+
+    }
+
```

Comment:
Some simple comments might make it easier to follow the simulation pane creation.